src/pages/index.js

```diff
@@ -55,7 +60,11 @@ const HomeSplash = props => {
 
   return (
     <SplashContainer>
-      <Logo img_src={`${baseUrl}img/logo-large.png`} />
+      <Logo
+        img_src={`${baseUrl}img/logo-large.png`}
+        img_width={128}
```

Comment:
I prefer refactoring all of these to be the same as the other component (camelCased) instead of adding more props that don't follow normal conventions. Wdyt?

Reply:
i make pr via local file conventions. because 1 pr = 1 feature. if you want, i can add refactoring, too. 
but it is not usual way. it is better to do new pr with refactoring (change to camel, and add linter to keep it in control)

Reply:
Since we're adding another prop here, I prefer seeing it done in this PR to follow the "boy scout rule". No need for a lint rule though :)

Reply:
fix only my new code, or small refactoring for all this part?

Reply:
"I prefer refactoring all of these" - ok

Reply:
make update at pr